front/pages/w/[wId]/builder/assistants/create.tsx

```diff
@@ -117,6 +129,50 @@ export default function CreateAssistant({
     );
   };
 
+  const tagsRefsMap = useRef<{
+    [key: string]: React.MutableRefObject<HTMLDivElement | null>;
+  }>({});
+
+  useEffect(() => {
+    templateTags.forEach((tag: string) => {
+      tagsRefsMap.current[tag] = tagsRefsMap.current[tag] || createRef();
+    });
+  }, [templateTags]);
+
+  const scrollToTag = (tagName: string) => {
+    const SCROLL_OFFSET = 64; // Header size
+    const scrollToElement = tagsRefsMap.current[tagName]?.current;
+    const scrollContainerElement = document.getElementById("main-content");
+
+    if (!scrollToElement || !scrollContainerElement) {
+      console.error("Element or scrollable container not found.", tagName);
```

Comment:
This really isn't actionable for the user so I would suggest to remove

Reply:
Yes definitely not actionable for the user, that's why I put it as console.error and not a Notification. 
I thought this could be helpful to debug for a developer in case the scrollContainerElement id changes for example, but I'm strongly convinced it's useful so I'll follow your reco and remove it 👍🏻